src/NUnitFramework/framework/Is.cs

```diff
@@ -486,5 +486,24 @@ public static RangeConstraint InRange(object from, object to)
 
         #endregion
 
+        #region Any Of
+
+        /// <summary>
+        /// Returns a constraint that tests if an item is equal to any of parameters
+        /// </summary>
+        /// <param name="expected">Expected values</param>
+        public static AnyOfConstraint AnyOf(params object[] expected)
+        {
+            if (expected.Length == 0)
+            {
+                throw new ArgumentException(nameof(expected),
+                    "Expected values are required for AnyOf constraint!");
```

Comment:
This might be optional if you add the guard to the constructor of the constraint. It would be better to use our Guard class for this in any case, just as a matter of clarity.